src/components/Overlay/Overlay.scss

```diff
@@ -15,6 +15,7 @@
   right:  0;
   top:    0;
   z-index: $ms-zIndex-Overlay;
+  cursor: pointer;
```

Comment:
Ummm I don't think we should do this. This will make all overlays look like a clickable target?